### PR TITLE
Use the upstream version of ffmuc-mesh-vpn-wireguard-vxlan

### PR DESCRIPTION
**Pull request by awlx**

### CODE REVIEW

File: domains/ffmuc_freising.conf [hunk 1]
```diff
@@ -51,6 +51,7 @@
       enabled = 'true',
       iface = 'wg_mesh_vpn',
       limit = '1', -- actually unused
+      broker = 'broker.ffmuc.net',
```
Comment (goligo) [inline domains/ffmuc_freising.conf]: This should also be 'broker.ffmuc.net/api/v1/wg/key/exchange', right?

Comment (awlx) [reply domains/ffmuc_freising.conf]: Yep

Comment (awlx) [inline domains/ffmuc_freising.conf]:
```suggestion
      broker = 'broker.ffmuc.net/api/v1/wg/key/exchange',
```

File: site.mk [hunk 2]
```diff
@@ -14,7 +14,7 @@ GLUON_FEATURES := \
 	ebtables-filter-ra-dhcp \
 	ebtables-source-filter \
 	mesh-batman-adv-15 \
-	mesh-vpn-wireguard-vxlan \
+	ffmuc-mesh-vpn-wireguard-vxlan \
```
Comment (krombel) [inline site.mk]: AFAIK this has to go into `GLUON_SITE_PACKAGES` and not in `GLUON_FEATURES`

**Review by krombel — DISMISSED** [review]

File: site.mk (hunk 2 above)
Comment (GoliathLabs) [reply site.mk]: @awlx after incorporating the changes suggested by @krombel everything should be good to go from my side

Comment (awlx) [reply site.mk]: @GoliathLabs feel free to push to this branch, I ain't got no time to work on it.

Comment (GoliathLabs) [reply site.mk]: @awlx @krombel should be good to go

**Review by GoliathLabs — APPROVED** [review]
Doesn't compile as expected

**Comment by goligo** [comment]
> LGTM

Just in case I missed something: You pushed a change which does not build, but nevertheless the change looks good to you and you approve the change to be merged?

**Comment by goligo** [comment]
The issue is that the package is named "ffmuc-**gluon**-mesh-vpn-wireguard-vxlan", but referenced as "ffmuc-mesh-vpn-wireguard-vxlan". From my perspective the package name should be changed in community-packages - the "gluon" does not belong in there.

**Comment by goligo** [comment]
I have created https://github.com/freifunkMUC/community-packages/pull/30 to rename the package - however I saw it has already been merged into gluon/community-packages with this name, so maybe we rather stick with it, instead of causing even more confusion?

File: site.mk [hunk 3]
```diff
@@ -36,6 +35,7 @@ GLUON_SITE_PACKAGES := \
 	ffho-ap-timer \
 	ffho-autoupdater-wifi-fallback \
 	ffho-web-ap-timer \
+	ffmuc-mesh-vpn-wireguard-vxlan \
```
Comment (awlx) [inline site.mk]:
```suggestion
	ffmuc-gluon-mesh-vpn-wireguard-vxlan \
```

**Comment by awlx** [comment]
@krombel @goligo @GoliathLabs good for review now. Node is online:
https://map.ffmuc.net/#!/en/map/5254003bc282

**Comment by GoliathLabs** [comment]
@goligo LGTM now :D

**Pull request merged**